data-prepper-plugins/ml-inference-processor/src/main/java/org/opensearch/dataprepper/plugins/ml/processor/util/RetryUtil.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.ml.processor.util;
+
+public class RetryUtil {
```

Comment:
Nit: You could put this in a common library for re-use between other plugins

Reply:
good point. Moved it to data-prepper-plugins:common